dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/DefaultQueryParser.java

```diff
@@ -83,6 +92,19 @@ public Query parse( Class<?> klass, List<String> filters, Junction.Type rootJunc
         return query;
     }
 
+    private Junction handleIdentifiablePath( Schema schema, String operator, Object arg ) {
+        Restriction displayNameRestriction = getRestriction( schema,"name", operator, arg );
```

Comment:
Are you sure that you should use name and not displayName here? what about translated names?